change/@fluentui-eslint-plugin-4301dbea-1be5-436e-bf0e-89ed473cd0b7.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Adding rnx-kit dependency and enabling the no-export-all rule for exporting library surfaces.",
+  "packageName": "@fluentui/eslint-plugin",
+  "email": "dzearing@microsoft.com",
+  "dependentChangeType": "patch"
```

Changes to eslint-plugin always need dependentChangeType: none to prevent spurious bumps of all packages
```suggestion
  "dependentChangeType": "none"
```

---

Should @fluentui/eslint-plugin then be marked as private, if it's not re-publishing on changes?

---

It should re-publish (`changeType: patch`) but does NOT need to make everything in our repo that depends on it bump